src/System.Numerics.Tensors/src/System/Numerics/Tensors/ICompressedSparseTensor.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Numerics.Tensors
+{
+    public interface ICompressedSparseTensor<T> : ITensor<T>
```

Comment:
We'll need to decide if we really want these derived interfaces.  I thought we'd prefer to not have them unless we could envision more than a single implementation of the derived shape.

Reply:
I think it is still worth exposing `ISparseTensor`, at the very least.

There are some concepts that should likely be exposed that are separate from `ITensor` and `IDenseTensor`

Reply:
That's reasonable.

Reply:
Removed the `ICompressedSparseTensor` interface.